CONTRIBUTING.md

```diff
@@ -46,13 +46,36 @@ See the [contributor's testing guide](https://github.com/kudobuilder/kudo/blob/m
 
 ## Community, Discussion, and Support
 
-Learn how to engage with the Kubernetes community on the [community page](https://kudo.dev/community/).
+Learn how to engage with the KUDO community on the [community page](https://kudo.dev/community/).
 
 You can reach the maintainers of this project at:
 
 - [Slack](https://kubernetes.slack.com/messages/kudo/)
 - [Mailing List](https://groups.google.com/d/forum/kudobuilder)
 
+## Code culture
+
+This is a set of practices we try to live by when developing KUDO. These are just defaults (soft rules). Deviations from them are possible, only has to be justified.
+
+### General guidelines
+- Master is always releasable (green CI)
+- All feature/bug-fixing work should have an open issue with a description
+- Every user-facing feature that is NOT behind a feature gate should have an integration test
+
+### Pull requests
+- One core-team member has to approve the PR to be able to merge (defined in code owners file)
+- When core-team member puts 'request for changes' on your PR, that stops merging until it's resolved (even when have enough approvals)
```

Comment:
a slightly different wording:
```suggestion
- One approval is enough to merge. However, if there are requests for change they have to be resolved prior to the merge
```

Reply:
There's also a different meaning behind the _slightly different wording_ though: do we block merging when anyone requests changes (_slightly different wording_ version), or only if a code owner requests changes (original version)?

Reply:
to be added in a future PR as @alenkacz recommended... but I would really like to encourage everyone to differentiate between "must change" and personal preferences.   It is a large grey area for sure... BUT... if
1. the feature is behind a feature gate or
2. this is a new feature that works, has tests, has no breaking changes and is green
that is the criteria! 

There is at least 1 small caveat which is the introduction of something as part of an API or otherwise public facing.

Reply:
I would argue that it doesn't matter. A "request for change" should be resolved in either case. If it's a soft/personal opinion/nit it can be a comment (rather than a request for change)

Reply:
btw. resolved != you have to do what the person requests. But a discussion is a good way to start - I just think we should have a way how to stop something from merging until that conversation happen :)

Reply:
💯 onboard with that. Sometimes "resolved" means "agree to disagree and move on".